priv/plts/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Does this file need to exist even with the changes to the top-level .gitignore?

Reply:
Either you use `.gitkeep` (a made-up thing) or preferably `.gitignore` as shown to keep the directory structure.

Maybe I don't, just following the pattern for years that I forgot if I must or not

Reply:
Ah ok it's just to keep the directory here like a gitkeep 👍